modules/aerodyn/src/AeroDyn_IO.f90

```diff
@@ -1786,19 +1878,19 @@ subroutine Calc_WriteOutput_BEMT
       end if
    
       ! Integrate force/moments over blades by performing mesh transfer to blade root points:
-!      do k=1,min(p%NumBlades,MaxBl)
-!         call Transfer_Line2_to_Point( y%BladeLoad(k), m%BladeRootLoad(k), m%B_L_2_R_P(k), ErrStat2, ErrMsg2, u%BladeMotion(k), u%BladeRootMotion(k) )
-!         ! Transform force vector to blade root coordinate system
-!         tmp = matmul( u%BladeRootMotion(k)%Orientation(:,:,1), m%BladeRootLoad(k)%force( :,1) )
-!         m%AllOuts( BAeroFx(k) ) = tmp(1)
-!         m%AllOuts( BAeroFy(k) ) = tmp(2)
-!         m%AllOuts( BAeroFz(k) ) = tmp(3)
-!         ! Transform moment vector to blade root coordinate system
-!         tmp = matmul( u%BladeRootMotion(k)%Orientation(:,:,1), m%BladeRootLoad(k)%moment( :,1) )
-!         m%AllOuts( BAeroMx(k) ) = tmp(1)
-!         m%AllOuts( BAeroMy(k) ) = tmp(2)
-!         m%AllOuts( BAeroMz(k) ) = tmp(3)
-!      end do  ! k=blades
+      do k=1,min(p%NumBlades,4)
```

Comment:
@ebranlard Why the loop to a minimum of 4 here? In other areas, the minimum is 3. Also, using `MaxBl` allows to change it all at once and prevent any inconsistencies.

Reply:
@andrew-platt added some clarity in that these outputs are specific to VAWTS. Without context, the difference in 3 and 4 is unclear, so a comment noting this would be helpful.

Reply:
The limit is purely arbitrary for now, and not limited to VAWT. I needed more outputs (actually I needed 9 blades), so I hardcoded outputs for an extra blade (without using the matlab script). Hannah's branch will have a better handling of the outputs, this is just temporary.